src/image/build-rpms.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Primary purpose of this script is to build the MicroShift RPMs and SRPMs
+# with adjusted version string.
+# Using `make {rpm,srpm}` it would hardcode a meaningless version string based on downstream git variables
+# which would not provide good information to identify the build and trace back its contents.
+# Following script overrides the version to include information about the downstream version and commit, and OKD tag.
+
+usage() {
+    echo "Script accepts at most one argument: all, rpm or srpm"
```

Comment:
I'm not a big fan of default arguments :). 

Can we make iit so that it's more "standard", like the following?

```
echo "Usage: $(basename "$0") <all | rpm | srpm>"
```